salt/modules/beacons.py

```diff
@@ -251,3 +251,93 @@ def disable(**kwargs):
             # Effectively a no-op, since we can't really return without an event system
             ret['comment'] = 'Event module not available. Beacons enable job failed.'
     return ret
+
+
+def enable_beacon(name, **kwargs):
+    '''
```

Comment:
Hrm...I guess I thought beacons were backported to 2015.5. I'll check that and change it if necessary. Nothing to do here, then. :)

Thanks!